console/console-init/ui/src/modules/iot-device/components/ReviewDevice/ReviewDevice.tsx

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import React from "react";
+import { IDeviceProp } from "modules/iot-device/containers/ReviewDeviceContainer";
+import { Title, Text, GridItem, Grid } from "@patternfly/react-core";
+import { StyleSheet, css } from "aphrodite";
+import { CredentialsView } from "modules/iot-device-detail";
+interface IReviewDeviceProps {
+  device?: IDeviceProp;
+  title?: string;
+}
+
+const style = StyleSheet.create({
+  font_size_20: {
+    fontSize: 20
```

Comment:
I don't think so we need to add font size, it should take default patternfly font size, if yoou need bigger size then use patternfly component like Title and use different sizes.